qiskit_ibm_runtime/transpiler/passes/scheduling/__init__.py

```diff
@@ -56,12 +56,12 @@
     from qiskit_ibm_runtime.transpiler.passes.scheduling import DynamicCircuitInstructionDurations
     from qiskit_ibm_runtime.transpiler.passes.scheduling import ALAPScheduleAnalysis
     from qiskit_ibm_runtime.transpiler.passes.scheduling import PadDelay
-    from qiskit_ibm_runtime.fake_provider import FakeJakarta
+    from qiskit_ibm_runtime.fake_provider import FakeJakartaV2
 
-    backend = FakeJakarta()
+    backend = FakeJakartaV2()
 
     # Temporary workaround for mock backends. For real backends this is not required.
-    backend.configuration().basis_gates.append("if_else")
+    backend._conf_dict['basis_gates'].append('if_else')
```

Comment:
This hack could be deleted when merging the [PR](https://github.com/Qiskit/qiskit-ibm-runtime/pull/1710).